src/interfaces/new-request-interface.ts

```diff
@@ -15,7 +15,6 @@ export interface NewRequestIF {
   analysisJSON: AnalysisJSONI
   applicant: ApplicantI
   assumedNameOriginal: string
-  businessLookup: BusinessLookupResultIF
```

Comment:
I think we don't need this in the store -- its lifespan is only while doing a business lookup on the Search page. After that, other variables are set from the data.